packages/office-ui-fabric-react/src/components/Facepile/Facepile.styles.ts

```diff
@@ -1,17 +1,154 @@
 import { IFacepileStyleProps, IFacepileStyles } from './Facepile.types';
 
-export const getStyles = (props: IFacepileStyleProps): IFacepileStyles => {
-  const { className } = props;
+import {
+  IStyle,
+  hiddenContentStyle,
+  getFocusStyle,
+  getGlobalClassNames,
+} from '../../Styling';
 
-  return {
+const GlobalClassNames = {
+  root: 'ms-Facepile',
+  addButton: 'ms-Facepile-addButton ms-Facepile-itemButton',
+  descriptiveOverflowButton: 'ms-Facepile-descriptiveOverflowButton ms-Facepile-itemButton',
+  itemButton: 'ms-Facepile-itemButton ms-Facepile-person',
+  itemContainer: 'ms-Facepile-itemContainer',
+  members: 'ms-Facepile-members',
+  overflowButton: 'ms-Facepile-overflowButton ms-Facepile-itemButton',
+};
+
+export const styles = (
+  props: IFacepileStyleProps
+): IFacepileStyles => {
+  const {
+    className,
+    theme,
+    spacingAroundItemButton = 2,
+  } = props;
+
+  const { palette, fonts } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const ItemButtonStyles: IStyle = {
+    textAlign: 'center',
+    padding: 0,
+    borderRadius: '50%',
+    verticalAlign: 'top',
+    selectors: {
+      'button&': {
+        display: 'inline',
+        background: 'none',
+        backgroundColor: 'transparent',
+        padding: 0,
+        cursor: 'pointer',
+        border: 'none',
+        selectors: {
+          '&::-moz-focus-inner': {
+            padding: 0,
+            border: 0
+          }
+        }
+      }
+    }
+  };
+
+  return ({
     root: [
-      'ms-Facepile',
+      classNames.root,
       {
-        // Insert css properties
+        width: 'auto'
       },
       className
-    ]
+    ],
+    addButton: [
+      classNames.addButton,
+      getFocusStyle(theme, -1),
+      ItemButtonStyles,
+      {
+        selectors: {
+          'button&': {
```

Comment:
>button& [](start = 11, length = 7)

this looks wrong to me... If this is intended, could you explain what you're doing here?

Reply:
(This could have been trying to preserve something from original styling.

Reply:
The original styling https://github.com/OfficeDev/office-ui-fabric-react/blob/b8ca29ca023b038681daa81b51bb52c927ca1545/packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss#L29  contained styles for both
```
.itemButton {
...
}

button.itemButton {
...
}
```
The `'button&:'` selector here is replicating that.